tests/ha/add_sbd_device.pm

```diff
@@ -0,0 +1,73 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Add a SBD device to Cluster
+# Maintainer: QE-SAP <qe-sap@suse.de>
+
+=head1 NAME
+
+ha/add_sbd_device.pm - Add a SBD device to Cluster
+
+=head1 DESCRIPTION
+
+This module is used to add a second SBD device to Cluster and check the result.
+
+The key tasks performed by this module include:
+
+=over
+
+=item * Add a SBD device on node01
+
+=item * Restart cluster to apply the change
+
+=item * Chech `crm sbd status` result on node01 and node02, there should be two sbd devices.
+
+=back
+
+This includes the lock for multi-machine test.
+
+=over 
+
+=item * C<CLUSTER_ADD_SBD_DEVICE_$cluster_name>
+
+=back
+
```

Comment:
What about adding a section about Variables, even if this test module seems only to use one: `HOSTNAME`